packages/core/src/Stores/client-store.js

```diff
@@ -2674,14 +2674,19 @@ export default class ClientStore extends BaseStore {
                 }
 
                 const data = await WS.send({ passkeys_list: 1 });
-                const should_show_effortless_login_modal =
-                    this.root_store.ui.is_mobile &&
-                    !data?.passkeys_list?.length &&
-                    this.is_passkey_supported &&
-                    show_effortless_login_modal &&
-                    this.is_logged_in;
-
-                this.setShouldShowEffortlessLoginModal(should_show_effortless_login_modal);
+
+                if (data?.passkeys_list) {
+                    const should_show_effortless_login_modal =
+                        this.root_store.ui.is_mobile &&
+                        !data?.passkeys_list?.length &&
```

Comment:
do we need thischeck now?

Reply:
@amina-deriv , it is more safe to have this check, as is there is no passkeyes returned, and data.passkeys_list.length in undefined, the calculated value will be true (!undefined)